intercom-java/src/main/java/io/intercom/api/User.java

```diff
@@ -55,13 +55,40 @@ public static User update(User user) throws InvalidException, AuthorizationExcep
         return DataResource.update(UserUpdate.buildFrom(user), "users", User.class);
     }
 
-    public static User delete(String id)
+    /**
+     * @deprecated  Replaced by {@link #archive(String)}. Renamed for consistency with API language
+     */
+    public static User delete(String id){
```

Comment:
Minor, formatting.

Reply:
Thanks for the review @eripe970. Not sure I get exactly which formatting that you're referring to here so would be great if I could get a bit more details.

Will change some the issues that I've responded to but will leave the naming convention to the other dev teammates to take a look at 😄

Reply:
Cool, normally it's a space before the {, like `delete(String id){` => `delete(String id) {`. At least I think that's the convention that is used in a lot of code-bases.

Reply:
Ah got it 👍  I do see the other files do have that space so added it in